code/src/components/Footer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components/macro';
+
+import { OuterFlexWrapper, InnerFlexWrapper } from '../styling/GlobalStyles';
+
+export const Footer = ({ setHistoryVisible }) => {
+  return (
+    <FooterWrapper>
+      <DirectionsButtonWrapper>
+        <FooterImageButton onClick={() => setHistoryVisible(true)}>
```

Comment:
Not sure what's supposed to happen when I click this, but I only see a blank, white screen

Reply:
Dang.. it's supposed to look like this.. 
<img width="735" alt="Skärmavbild 2020-12-28 kl  18 07 13" src="https://user-images.githubusercontent.com/47143197/103231234-9e222980-4937-11eb-95b3-bb4fe4f7b85d.png">